layouts/partials/common/nav.html

```diff
@@ -14,7 +14,7 @@
 <nav id="al-primary-navbar" class="al-primary-navbar navbar p-0 sticky-top navbar-expand-lg navbar-dark flex-column with-motd">
 
     <div class="container  my-3">
-        <a class="navbar-brand" href="/">
+        <a class="navbar-brand" href="{{ "" | relLangURL }}">
```

Comment:
@codyro :point_up:

Reply:
I was saying this should solve what you are referring to. Am I missing something?

Reply:
🤦 I was testing in the wrong tab. I am so, so sorry.